Tests/AK/TestIPv6Address.cpp

```diff
@@ -105,14 +105,17 @@ TEST_CASE(should_make_empty_optional_from_out_of_range_values)
     EXPECT(!addr.has_value());
```

Comment:
Commit message: "that object" -> "that objects"

Additionally, the test also asserts that an object is equal to itself.

Reply:
Fixed the first part.

Not sure what you mean by the second sentence. The commit doesn't add any test that asserts and object is equal to itself, only that two objects with containing the same inet6-address are equal. Is the sentence about the commit-message, a desire to add an extra assertion, or something else?

Reply:
Your commit message states:

> The existing test checks that objects with different inet6-addresses are unequal.

However, the existing test does not only check this, it also checks whether the same object is equal to itself. I feel that this is what you expanded upon: instead of comparing an object with itself for equality, you compare two different objects for equality.

Reply:
What I'm asking you to do, is to change the commit message to either leave that part out (we can see the existing tests) or to explain that you're expanding on the equality comparison by comparing to a different object instead of the same.

Reply:
Ah, I see. I just left that out then :) Thanks for the other comments as well :)